keyboards/kegen/gboy/info.json

```diff
@@ -0,0 +1,85 @@
+{
+    "keyboard_name": "Kegen G-Boy", 
+    "url": "", 
+    "maintainer": "@mattchan", 
+    "layouts": {
```

Comment:
```suggestion
    "usb": {
        "vid": "0x4B65",
        "pid": "0x6762",
        "device_version": "0.0.1"
    },
    "layouts": {
```

Reply:
Why are these values moved here instead of being defined in config.h?

Reply:
Nevermind, I found the answer in the QMK Discord server and found the relevant documentation.